fdbclient/NativeAPI.actor.cpp

```diff
@@ -3216,10 +3216,12 @@ ACTOR Future<std::vector<std::pair<KeyRange, BlobWorkerInterface>>> getBlobGranu
     KeyRange keys,
     int limit,
     Reverse reverse,
+    bool justGranules,
```

Comment:
I'd use `FDB_BOOL_PARAM` for this, much like `UseTenant` or `Reverse`

Reply:
Sounds good. Since Evan approved and it's tough to get CI passing, and I have another follow up PR here anyway, I'll include those changes there.